simGL.egg-info/PKG-INFO

```diff
@@ -0,0 +1,16 @@
+Metadata-Version: 1.1
```

Comment:
The contents of the simGL.egg-info package should be committed into the repository. It's probably a good idea to add this folder to the `.gitignore` file.